app/models/concerns/rad_user.rb

```diff
@@ -133,11 +132,6 @@ def admin?
     permission?(:admin)
   end
 
-  def auto_approve?
```

Comment:
this feature is no longer used and just getting it out of the way to avoid confusion, we can easily add it back or something like it if needed